.storybook/preview.js

```diff
@@ -11,7 +11,7 @@ import MetaMetricsProviderStorybook from './metametrics';
 import testData from './test-data.js';
 import { Router } from 'react-router-dom';
 import { createBrowserHistory } from 'history';
-import { _setBackgroundConnection } from '../ui/store/actions';
+import { _setBackgroundConnection } from '../ui/store/action-queue';
```

Comment:
NIT: my brain sees underscore before a variable/method and it thinks `private` variable/method.

Reply:
I would avoid changing existing method in this PR.